kuma/wiki/templates/wiki/feed_docs_updated.html

```diff
@@ -0,0 +1,12 @@
+<p>
+  <a href="{{ obj.parent.get_absolute_url()|absolutify }}" title="{{ obj.parent.title }}">
+    {% trans locale=obj.parent.locale %}View {locale} parent{% endtrans %}
+  </a>
+  (<a href="{{ mod_url|absolutify }}">{% trans modified=obj.parent.modified %}last modified at {modified}{% endtrans %}</a>)
+</p>
+<p>
+  <a href="{{ add_utm(obj.get_edit_url()|absolutify, 'feed', medium='format') }}" title="{{ obj.title }}">
```

Comment:
ah ok. When I used `medium=format` nothing ended up in the URL for the utm_medium query string. I'll use `rss` since that's what this used in.